source/i18n/en-US.ts

```diff
@@ -1,4 +1,76 @@
+import { textJoin } from 'mobx-i18n';
+import enUS_2017 from '../page/2017/i18n/en-US';
 export default {
```

Comment:
```suggestion
import { textJoin } from 'mobx-i18n';

import enUS_2017 from '../page/2017/i18n/en-US';

 export default {
```
导入内外部模块和导出之间都空一行会比较清晰。